packages/paste-icons/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "parser": "babel-ts",
```

Comment:
Note: We need a specific .prettierrc file just for icons because svg-to-react queries it and it needs the parser.

I tried adding just the parser to the global prettierrc, but that caused problems because not all files in paste are typescript (ex. js, json).

Reply:
I wanted to add this info as a comment in the file itself but we don't allow comments in JSON :///